--- partimage-0.6.8/src/client/netclient.h.orig	2010-04-14 23:09:04.205217015 +0200
+++ partimage-0.6.8/src/client/netclient.h	2010-04-14 23:09:12.318548298 +0200
@@ -35,7 +35,7 @@ private:
 #ifdef HAVE_SSL
   SSL_CTX * ctx;
   X509 * server_cert;
-  SSL_METHOD * meth;
+  const SSL_METHOD * meth;
 #endif
   bool m_bUseSSL;
   bool m_bMustLogin;
--- partimage-0.6.8/src/server/netserver.h.orig	2010-04-14 23:14:09.311851072 +0200
+++ partimage-0.6.8/src/server/netserver.h	2010-04-14 23:14:15.971852561 +0200
@@ -41,7 +41,7 @@ private:
 #ifdef HAVE_SSL
   SSL_CTX * ctx;
   X509 * client_cert;
-  SSL_METHOD * meth; 
+  const SSL_METHOD * meth; 
   int err;
 #endif
 
