diff -urN partimage-0.3.5.3.org/Makefile.am partimage-0.3.5.3/Makefile.am
--- partimage-0.3.5.3.org/Makefile.am	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/Makefile.am	Sun Apr 29 18:55:14 2001
@@ -1,32 +1,26 @@
 ####### kdevelop will overwrite this part!!! (begin)##########
 
-SUBDIRS = partimage po
+SUBDIRS = partimage intl po
 EXTRA_DIST = partimage.kdevprj AUTHORS COPYING ChangeLog INSTALL README TODO partimage.lsm 
 
 install-data-local:
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) AUTHORS /usr/share/doc/partimage/AUTHORS
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) COPYING /usr/share/doc/partimage/COPYING
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) ChangeLog /usr/share/doc/partimage/ChangeLog
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) INSTALL /usr/share/doc/partimage/INSTALL
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) README /usr/share/doc/partimage/README
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) TODO /usr/share/doc/partimage/TODO
-	$(mkinstalldirs) /usr/share/doc/partimage/
-	$(INSTALL_DATA) partimage.lsm /usr/share/doc/partimage/partimage.lsm
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/doc/partimage/
+	$(INSTALL_DATA) AUTHORS $(DESTDIR)$(datadir)/doc/partimage/AUTHORS
+	$(INSTALL_DATA) COPYING $(DESTDIR)$(datadir)/doc/partimage/COPYING
+	$(INSTALL_DATA) ChangeLog $(DESTDIR)$(datadir)/doc/partimage/ChangeLog
+	$(INSTALL_DATA) INSTALL $(DESTDIR)$(datadir)/doc/partimage/INSTALL
+	$(INSTALL_DATA) README $(DESTDIR)$(datadir)/doc/partimage/README
+	$(INSTALL_DATA) TODO $(DESTDIR)$(datadir)/doc/partimage/TODO
+	$(INSTALL_DATA) partimage.lsm $(DESTDIR)$(datadir)/doc/partimage/partimage.lsm
 
 uninstall-local:
-	-rm -f /usr/share/doc/partimage/AUTHORS
-	-rm -f /usr/share/doc/partimage/COPYING
-	-rm -f /usr/share/doc/partimage/ChangeLog
-	-rm -f /usr/share/doc/partimage/INSTALL
-	-rm -f /usr/share/doc/partimage/README
-	-rm -f /usr/share/doc/partimage/TODO
-	-rm -f /usr/share/doc/partimage/partimage.lsm
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/AUTHORS
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/COPYING
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/ChangeLog
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/INSTALL
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/README
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/TODO
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/partimage.lsm
 
 ####### kdevelop will overwrite this part!!! (end)############
 AUTOMAKE_OPTIONS = foreign
diff -urN partimage-0.3.5.3.org/configure.in partimage-0.3.5.3/configure.in
--- partimage-0.3.5.3.org/configure.in	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/configure.in	Sun Apr 29 19:01:40 2001
@@ -16,6 +16,9 @@
 AC_PROG_INSTALL
 AC_PROG_LN_S
 
+ALL_LINGUAS="de fr gl it ko"
+AM_GNU_GETTEXT
+
 dnl Checks for libraries.
 AC_CHECK_LIB(bz2, BZ2_bzopen, [],	AC_MSG_ERROR([*** bzip2 library (libbz2) not found or too old: version 1.0.0 or more recent is need]))
 AC_CHECK_LIB(com_err, com_err, [],	AC_MSG_ERROR([*** com_err library (libcom_err) not found]))
@@ -27,4 +30,4 @@
 AC_HEADER_STDC
 AC_CHECK_HEADERS(fcntl.h unistd.h)
 
-AC_OUTPUT(Makefile partimage/Makefile partimage/docs/Makefile partimage/docs/en/Makefile po/Makefile )
+AC_OUTPUT(Makefile partimage/Makefile partimage/docs/Makefile partimage/docs/en/Makefile intl/Makefile po/Makefile.in )
diff -urN partimage-0.3.5.3.org/partimage/Makefile.am partimage-0.3.5.3/partimage/Makefile.am
--- partimage-0.3.5.3.org/partimage/Makefile.am	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/partimage/Makefile.am	Sun Apr 29 19:07:13 2001
@@ -1,5 +1,5 @@
 ####### kdevelop will overwrite this part!!! (begin)##########
-bin_PROGRAMS = partimage
+sbin_PROGRAMS = partimage
 partimage_SOURCES = chpfspart.cpp parttable.cpp gui_text.cpp creiserpart.cpp imagefile.cpp cntfspart.cpp misc.cpp cfatpart.cpp common.cpp cext2part.cpp main.cpp 
 partimage_LDADD   = -lext2fs -lcom_err -lz -lbz2 -lnewt -lslang
 
diff -urN partimage-0.3.5.3.org/partimage/docs/en/Makefile.am partimage-0.3.5.3/partimage/docs/en/Makefile.am
--- partimage-0.3.5.3.org/partimage/docs/en/Makefile.am	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/partimage/docs/en/Makefile.am	Sun Apr 29 18:48:30 2001
@@ -4,46 +4,34 @@
 EXTRA_DIST = index.html index-1.html index-2.html index-3.html index-4.html index-5.html index-6.html index-7.html index-8.html index.sgml screenshot-001.png screenshot-002.png screenshot-003.png 
 
 install-data-local:
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index.html /usr/share/doc/partimage/html/index.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-1.html /usr/share/doc/partimage/html/index-1.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-2.html /usr/share/doc/partimage/html/index-2.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-3.html /usr/share/doc/partimage/html/index-3.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-4.html /usr/share/doc/partimage/html/index-4.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-5.html /usr/share/doc/partimage/html/index-5.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-6.html /usr/share/doc/partimage/html/index-6.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-7.html /usr/share/doc/partimage/html/index-7.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index-8.html /usr/share/doc/partimage/html/index-8.html
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) index.sgml /usr/share/doc/partimage/html/index.sgml
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) screenshot-001.png /usr/share/doc/partimage/html/screenshot-001.png
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) screenshot-002.png /usr/share/doc/partimage/html/screenshot-002.png
-	$(mkinstalldirs) /usr/share/doc/partimage/html/
-	$(INSTALL_DATA) screenshot-003.png /usr/share/doc/partimage/html/screenshot-003.png
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/doc/partimage/html/
+	$(INSTALL_DATA) index.html $(DESTDIR)$(datadir)/doc/partimage/html/index.html
+	$(INSTALL_DATA) index-1.html $(DESTDIR)$(datadir)/doc/partimage/html/index-1.html
+	$(INSTALL_DATA) index-2.html $(DESTDIR)$(datadir)/doc/partimage/html/index-2.html
+	$(INSTALL_DATA) index-3.html $(DESTDIR)$(datadir)/doc/partimage/html/index-3.html
+	$(INSTALL_DATA) index-4.html $(DESTDIR)$(datadir)/doc/partimage/html/index-4.html
+	$(INSTALL_DATA) index-5.html $(DESTDIR)$(datadir)/doc/partimage/html/index-5.html
+	$(INSTALL_DATA) index-6.html $(DESTDIR)$(datadir)/doc/partimage/html/index-6.html
+	$(INSTALL_DATA) index-7.html $(DESTDIR)$(datadir)/doc/partimage/html/index-7.html
+	$(INSTALL_DATA) index-8.html $(DESTDIR)$(datadir)/doc/partimage/html/index-8.html
+	$(INSTALL_DATA) index.sgml $(DESTDIR)$(datadir)/doc/partimage/html/index.sgml
+	$(INSTALL_DATA) screenshot-001.png $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-001.png
+	$(INSTALL_DATA) screenshot-002.png $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-002.png
+	$(INSTALL_DATA) screenshot-003.png $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-003.png
 
 uninstall-local:
-	-rm -f /usr/share/doc/partimage/html/index.html
-	-rm -f /usr/share/doc/partimage/html/index-1.html
-	-rm -f /usr/share/doc/partimage/html/index-2.html
-	-rm -f /usr/share/doc/partimage/html/index-3.html
-	-rm -f /usr/share/doc/partimage/html/index-4.html
-	-rm -f /usr/share/doc/partimage/html/index-5.html
-	-rm -f /usr/share/doc/partimage/html/index-6.html
-	-rm -f /usr/share/doc/partimage/html/index-7.html
-	-rm -f /usr/share/doc/partimage/html/index-8.html
-	-rm -f /usr/share/doc/partimage/html/index.sgml
-	-rm -f /usr/share/doc/partimage/html/screenshot-001.png
-	-rm -f /usr/share/doc/partimage/html/screenshot-002.png
-	-rm -f /usr/share/doc/partimage/html/screenshot-003.png
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-1.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-2.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-3.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-4.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-5.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-6.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-7.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index-8.html
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/index.sgml
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-001.png
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-002.png
+	-rm -f $(DESTDIR)$(datadir)/doc/partimage/html/screenshot-003.png
 
 ####### kdevelop will overwrite this part!!! (end)############
diff -urN partimage-0.3.5.3.org/po/POTFILES.in partimage-0.3.5.3/po/POTFILES.in
--- partimage-0.3.5.3.org/po/POTFILES.in	Thu Jan  1 01:00:00 1970
+++ partimage-0.3.5.3/po/POTFILES.in	Sun Apr 29 18:58:06 2001
@@ -0,0 +1,15 @@
+# List of files which contain translatable strings.
+
+partimage/gui_text.cpp
+partimage/cext2part.cpp
+partimage/parttable.cpp
+partimage/main.cpp
+partimage/chpfspart.cpp
+partimage/misc.cpp
+partimage/imagefile.cpp
+partimage/cfatpart.cpp
+partimage/creiserpart.cpp
+partimage/common.cpp
+partimage/cntfspart.cpp
+bootroot2cd.cpp
+
diff -urN partimage-0.3.5.3.org/po/de.po partimage-0.3.5.3/po/de.po
--- partimage-0.3.5.3.org/po/de.po	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/po/de.po	Sun Apr 29 19:02:53 2001
@@ -12,7 +12,7 @@
 "Last-Translator: alex kern <alex.kern@gmx.de>\n"
 "Language-Team: germany <de@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ISO 8859-1\n"
+"Content-Type: text/plain; charset=ISO8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: KBabel 0.6\n"
 
diff -urN partimage-0.3.5.3.org/po/fr.po partimage-0.3.5.3/po/fr.po
--- partimage-0.3.5.3.org/po/fr.po	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/po/fr.po	Sun Apr 29 19:03:00 2001
@@ -9,7 +9,7 @@
 "Last-Translator: François Dupoux <dupoux@club-internet.fr>\n"
 "Language-Team:  <de@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ISO 8859-1\n"
+"Content-Type: text/plain; charset=ISO8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: KBabel 0.5.3\n"
 
diff -urN partimage-0.3.5.3.org/po/gl.po partimage-0.3.5.3/po/gl.po
--- partimage-0.3.5.3.org/po/gl.po	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/po/gl.po	Sun Apr 29 19:03:40 2001
@@ -10,7 +10,7 @@
 "Last-Translator: Sergio Rua <srua@gpul.org>\n"
 "Language-Team: Galician <trasno@gpul.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=iso-8859-1\n"
+"Content-Type: text/plain; charset=iso8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #. standard infos
diff -urN partimage-0.3.5.3.org/po/it.po partimage-0.3.5.3/po/it.po
--- partimage-0.3.5.3.org/po/it.po	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/po/it.po	Sun Apr 29 19:03:31 2001
@@ -10,7 +10,7 @@
 "Last-Translator: François Dupoux <dupoux@club-internet.fr>\n"
 "Language-Team:  <de@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ISO 8859-1\n"
+"Content-Type: text/plain; charset=ISO8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: KBabel 0.5.3\n"
 
diff -urN partimage-0.3.5.3.org/po/ko.po partimage-0.3.5.3/po/ko.po
--- partimage-0.3.5.3.org/po/ko.po	Sun Apr 29 18:45:14 2001
+++ partimage-0.3.5.3/po/ko.po	Sun Apr 29 19:04:00 2001
@@ -1409,4 +1409,3 @@
 #: ../partimage/chpfspart.cpp:315 ../partimage/creiserpart.cpp:374
 msgid "writing header"
 msgstr "Çì´õ¸¦ ±â·ÏÇÏ´Â ÁßÀÔ´Ï´Ù."
- 
\ No newline at end of file
