diff -urNp -x '*.orig' partimage-0.6.9.org/src/client/main.cpp partimage-0.6.9/src/client/main.cpp
--- partimage-0.6.9.org/src/client/main.cpp	2010-07-25 17:30:31.000000000 +0200
+++ partimage-0.6.9/src/client/main.cpp	2021-10-04 22:06:07.562637662 +0200
@@ -83,6 +83,7 @@ static struct option const long_options[
   {"fully-batch", required_argument, NULL, 'B'},
   {"nocheck", no_argument, NULL, 'c'},
   {"nodesc", no_argument, NULL, 'd'},
+  {"description",required_argument, NULL, 'D'},
   {"erase", no_argument, NULL, 'e'},
   {"finish", required_argument, NULL, 'f'},
   {"debug", required_argument, NULL, 'g'},
@@ -110,7 +111,7 @@ static struct option const long_options[
 
 static char finish_last_countfile_lock[]="/tmp/partimageXXXXXX.lock";
 static char finish_last_countfile[]="/tmp/partimageXXXXXX";
-static char optstring[]="z:oV:ecmdhf:s:p:bwg:vyLnSMa:iU:P:XB:";
+static char optstring[]="z:oV:ecmdhf:s:p:bwg:vynSMa:iU:P:XB:D:";
 FILE * g_fDebug; // debug file
 FILE * g_fLocalDebug; // debug file
 CInterface * g_interface;
@@ -324,6 +325,12 @@ int main(int argc, char *argv[])
 	  case 'd': // don't ask any description
 	    options.bAskDesc = false;
 	    break;
+
+	  case 'D': // description in batch mode
+	    strncpy(options.szDescription, optarg, MAX_DESCRIPTION);
+	    *(options.szDescription+MAX_DESCRIPTION-1) = '\0';
+	    options.bAskDesc = false;
+	    break;
 	  
 	  case 'h': // help
 	    usage();
@@ -1013,6 +1020,7 @@ void usage()
 	      "* -c,  --nocheck       don't check the partition before saving\n"
 	      "* -o,  --overwrite     overwrite the existing image file without confirmation\n"
 	      "* -d,  --nodesc        don't ask any description for the image file\n"
+	      "* -DX, --description=X set image description in batch mode\n"
 	      "* -V,  --volume        (split image into multiple volumes files)\n"
 	      "  -VX, --volume=X      create volumes with a size of X MiB\n"
 	      "* -w,  --waitvol       wait for a confirmation after each volume change\n"
diff -urNp -x '*.orig' partimage-0.6.9.org/src/client/misc.cpp partimage-0.6.9/src/client/misc.cpp
--- partimage-0.6.9.org/src/client/misc.cpp	2021-10-04 22:06:07.452637694 +0200
+++ partimage-0.6.9/src/client/misc.cpp	2021-10-04 22:06:07.562637662 +0200
@@ -437,6 +437,10 @@ void savePartition(char *szDevice, char
       g_interface -> askDescription(headMain.szPartDescription,
 				    MAX_DESCRIPTION);
     }
+  else if (strlen(options->szDescription)>0)
+    {
+      strcpy(headMain.szPartDescription,options->szDescription);
+    }
 
   // bzip2 bug workaround
   if ((options->dwCompression == COMPRESS_BZIP2) 
diff -urNp -x '*.orig' partimage-0.6.9.org/src/shared/common.h partimage-0.6.9/src/shared/common.h
--- partimage-0.6.9.org/src/shared/common.h	2010-07-25 17:30:31.000000000 +0200
+++ partimage-0.6.9/src/shared/common.h	2021-10-04 22:06:07.562637662 +0200
@@ -143,6 +143,7 @@ struct COptions
   char szUserName[MAX_USERNAMELEN];
   char szPassWord[MAX_PASSWORDLEN];
   char szFullyBatchMode[2048];
+  char szDescription[MAX_DESCRIPTION];
 };
 
 #define MAX_UNAMEINFOLEN 65 //SYS_NMLN
