--- partimage-0.6.8/src/client/misc.cpp.old	2009-10-13 15:08:25.000000000 +0200
+++ partimage-0.6.8/src/client/misc.cpp	2009-10-13 15:09:02.000000000 +0200
@@ -1923,8 +1923,8 @@
     }
   *nPartNum = atoi(cTemp);
   
-  showDebug(9, "entry found: %s %d %d %d %d\n", szDevice, nMajor, nMinor,
-    nBlocks, nPartNum); 
+  showDebug(9, "entry found: %s %d %d %d %d\n", szDevice, *nMajor, *nMinor,
+    *nBlocks, *nPartNum); 
  
   return cPtr;
 }
