diff -U 3 -H -d -r -N -- partimage-0.6.5_beta2.orig/src/client/fs/fs_jfs.h partimage-0.6.5_beta2/src/client/fs/fs_jfs.h
--- partimage-0.6.5_beta2.orig/src/client/fs/fs_jfs.h	2005-05-22 11:12:56.000000000 +0200
+++ partimage-0.6.5_beta2/src/client/fs/fs_jfs.h	2005-08-09 13:22:50.000000000 +0200
@@ -17,6 +17,7 @@
 #ifndef FS_JFS_H
 #define FS_JFS_H
 
+#include <stdint.h>
 #include "partimage.h"
 #include "common.h"
 #include "cbitmap.h"
@@ -34,14 +35,14 @@
 #define JFS_MAGIC_STRING      "JFS1"  // Magic u16: Version 1 
 #define JFS_SUPER_MAGIC       0x3153464A
 
-#define int16 signed short int
-#define uint16 unsigned short int
-#define int32 signed long int
-#define uint32 unsigned long int
-#define int64  signed long long int
-#define uint64  unsigned long long int
-#define int8  signed char
-#define uint8  unsigned char
+#define int16 int16_t
+#define uint16 uint16_t
+#define int32 int32_t
+#define uint32 uint32_t
+#define int64 int64_t
+#define uint64 uint64_t
+#define int8 int8_t
+#define uint8 uint8_t
 
 /*#define s16 signed short int
 #define u16 unsigned short int
diff -U 3 -H -d -r -N -- partimage-0.6.5_beta2.orig/src/client/fs/fs_ufs.h partimage-0.6.5_beta2/src/client/fs/fs_ufs.h
--- partimage-0.6.5_beta2.orig/src/client/fs/fs_ufs.h	2005-05-22 11:12:56.000000000 +0200
+++ partimage-0.6.5_beta2/src/client/fs/fs_ufs.h	2005-08-09 13:24:28.000000000 +0200
@@ -21,6 +21,7 @@
 #include "fs_base.h"
 
 #include <sys/types.h>
+#include <stdint.h>
 
 // ================================================
 // ================================================
--- partimage-0.6.5_beta2/src/client/fs/fs_xfs.h.orig	2005-09-22 22:36:45.000000000 +0200
+++ partimage-0.6.5_beta2/src/client/fs/fs_xfs.h	2005-09-22 22:37:05.000000000 +0200
@@ -19,6 +19,7 @@
 
 #include <ctype.h>
 #include <stdint.h>
+#include <asm/types.h>
 
 #include "partimage.h"
 #include "common.h"
@@ -149,9 +150,6 @@
 typedef	__uint64_t	xfs_dfiloff_t;	// block number in a file 
 typedef	__uint64_t	xfs_dfilblks_t;	// number of blocks in a file 
 
-typedef uint64_t __u64;
-typedef int64_t  __s64;
-
 typedef __u64	xfs_off_t;
 //typedef __s32	xfs32_off_t;
 typedef __u64	xfs_ino_t;	// <inode> type 
